cli/crates/server/src/errors.rs

```diff
@@ -220,6 +220,10 @@ pub enum UdfBuildError {
     #[error("could not link to file: {0}")]
     SymlinkFailure(IoError),
 
+    /// returned if kv data path is invalid. E.g: has non unicode characters
+    #[error("invalid KV data path: {0}")]
```

Comment:
We could add a hint for this that we don't support non-unicode paths, but not urgent if you don't want to add it here